HISTORY.md

```diff
@@ -0,0 +1,119 @@
+# Changelog
+
+## Next Version
+
+* Fix broken namespecific rclink ([#209](https://github.com/luxflux/puppet-openvpn/pull/209))
+
+## 4.0.1
```

Comment:
I'm not certain, but you might have to change this line to
`## 4.0.1 (2016-09-25)` for the autodetection of the `since_tag` to work correctly.

(If you're really bored, you can back fill the release dates of the other tags too).